src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/ConditionModalForm.jsx

```diff
@@ -27,12 +26,35 @@ class ConditionModalForm extends Component {
   }
 
   componentDidMount() {
-    const subflowOptions = this.props.subflows
+    const getFlowLabel = name => {
+      switch (name) {
```

Comment:
We try to avoid switch statements because it adds a lot of unnecessary boilerplate, please use an if/else statement instead